src/components/Charts/SparklineChart.tsx

```diff
@@ -114,7 +114,7 @@ export class SparklineChart extends React.Component<Props, State> {
         domainPadding={{ y: 1 }}
       >
         <ChartAxis tickCount={15} style={hiddenAxisStyle} />
-        <ChartAxis dependentAxis={true} style={hiddenAxisStyle} />
+        <ChartAxis tickCount={2} dependentAxis={true} />
```

Comment:
This component is used also in the graph sidepanel, worth to check that we are not introduced side effects.

Reply:
I guess this is due this change:
![image](https://user-images.githubusercontent.com/1662329/132857286-8858e45f-245b-42c0-b047-0e862eacd31f.png)
(A minor empty y-axis)

Not a big deal, but I wonder if it could be better just to pass a property to differentiate one use or another.

Reply:
Sure, sorry about that, I prefer to differentiate them, thanks @lucasponce